### PR TITLE
Add configuration capability back to multitool

### LINKED ISSUE #16241: Combine device configurator into multitool
I think this is the preference as otherwise multitool would mostly be used for antag activities (hacking).

### PR DESCRIPTION
**Pull request by juliangiebel**
## About the PR
As mentioned in this issue #16241, this PR adds the linking and list functionality back to the multitool.
Further changes like making the multitool rarer or adding a dedicated probing tool can be done in a later PR.

Fixes #16241

### CODE REVIEW

File: Resources/Prototypes/Catalog/Fills/Items/toolboxes.yml [hunk 1]
```diff
@@ -34,6 +34,8 @@
       - id: CableMVStack
       - id: trayScanner
         prob: 0.9
+      - id: NetworkConfigurator
```
Comment (AJCM-git) [inline Resources/Prototypes/Catalog/Fills/Items/toolboxes.yml:37]: the toolbox has a capacity of 60 that gets breached by 5 in the rare ocassion everything spawns

File: Resources/Prototypes/Catalog/VendingMachines/Inventories/youtool.yml [hunk 2]
```diff
@@ -8,6 +8,7 @@
     Wrench: 5
     Screwdriver: 5
     trayScanner: 5
+    NetworkConfigurator: 3
```
Comment (AJCM-git) [inline Resources/Prototypes/Catalog/VendingMachines/Inventories/youtool.yml:11]: you need to add it to the restock too

**Review by AJCM-git — CHANGES_REQUESTED** [review]

File: Resources/Prototypes/Catalog/Fills/Items/toolboxes.yml (hunk 1 above)
Comment (juliangiebel) [reply Resources/Prototypes/Catalog/Fills/Items/toolboxes.yml:37]: Should I or group it with the t-ray scanner?

Comment (AJCM-git) [reply Resources/Prototypes/Catalog/Fills/Items/toolboxes.yml:37]: idk, this toolbox is more for messing with power so a tray makes way more sense, i would just not put it in there

**Pull request merged**